### PR TITLE
Integrate with steps API [CIVIL-408]

### PR DESCRIPTION
**Pull request by tobek**

### CODE REVIEW

File: packages/newsroom-signup/src/DataWrapper.tsx [hunk 1]
```diff
@@ -81,16 +71,25 @@ export class DataWrapper extends React.Component<DataWrapperProps> {
...
-                              grantApproved,
-                              newsroomDeployTx,
-                              newsroomAddress,
-                              profileWalletAddress: data.currentUser.ethAddress,
-                              persistedCharter,
-                              saveAddress,
-                              persistCharter: this.saveCharterFuncFromMutation(saveCharter),
-                            });
+                            return (
+                              <Mutation mutation={saveStepsMutation}>
```
Comment (tobek) [inline packages/newsroom-signup/src/DataWrapper.tsx]: This now has 2 nested <Query>'s and 3 nested <Mutations>'s, needs refactoring with apollo's compose and HOCs, will do this after launch.

Comment (walfly) [reply packages/newsroom-signup/src/DataWrapper.tsx]: 👍

File: packages/newsroom-signup/src/ApplyToTCR/index.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ export interface ApplyToTCRStepOwnProps {
   address?: EthAddress;
   newsroom: any;
   civil?: Civil;
-  navigate(go: 1 | -1): void;
+  navigate(go: 1 | -1, allDone?: boolean): void;
```
Comment (walfly) [inline packages/newsroom-signup/src/ApplyToTCR/index.tsx]: can't devine this with like `step === numsteps`?

Comment (walfly) [reply packages/newsroom-signup/src/ApplyToTCR/index.tsx]: or `step > numsteps`

Comment (tobek) [reply packages/newsroom-signup/src/ApplyToTCR/index.tsx]: Yes, you are absolutely right (can just check `step = STEP.APPLIED`, simplifies the hack a bit.

**Pull request merged**